source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
 		"kb:shift+tab":None,
 	}
 
+class WordChart(Window):
+	def __init__(self,windowHandle, wordApplicationObject, wordChartObject,keyIndex=0):
+		self.windowHandle=windowHandle
+		self.wordApplicationObject=wordApplicationObject
+		self.wordChartObject=wordChartObject
+		self.currentSeriesIndex=0
+		self.keyIndex=keyIndex
+		self.chartElements={}
+		self.keyList=[]
+		try:
+			seriesCount=self.wordChartObject.SeriesCollection().Count
+		except:
+			seriesCount=None
+		if seriesCount:
+			for i in xrange(seriesCount):
+				key='series'+str(i+1)
+				self.chartElements[key]=(self.focusChartSeries,i+1)
+				self.keyList.append(key)
+				try:
+					trendlinesCount = self.wordChartObject.SeriesCollection(i+1).Trendlines().Count
+				except:
+					trendlinesCount = None
+				if trendlinesCount>=1:
+					key=key+"Trendline"
+					self.chartElements[key]=(self.focusSeriesTrendline,i+1)
+					self.keyList.append(key)
+		self.chartElements['otherElements']=(self.focusChartElements, None)
+		self.keyList.append('otherElements')
+		self.elementsCount=len(self.keyList)
+		super(WordChart,self).__init__(windowHandle=windowHandle)
+
+	def focusSeriesTrendline(self, seriesIndex):
+		obj=WordChartSeriesTrendline(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex, seriesIndex=seriesIndex)
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def focusChartSeries(self, seriesIndex):
+		self.currentSeriesIndex=seriesIndex
+		obj=WordChartSeries(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex, seriesIndex=self.currentSeriesIndex)
+		self.wordChartObject.SeriesCollection(self.currentSeriesIndex).Select()
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def focusChartElements(self):
+		obj=WordChartElement(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex)
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def _get_name(self):
+		if self.wordChartObject.HasTitle:
+			name=self.wordChartObject.ChartTitle.Text
+		else:
+			name=self.wordChartObject.Name
+		#find the type of the chart
+		chartType = self.wordChartObject.ChartType
+		chartTypeText = _chartConstants.chartTypeDict.get(chartType,
+                # Translators: Reported when the type of a chart is not known.
+                                _("unknown"))
+		# Translators: Message reporting the title and type of a chart.
+		text=_("Chart title: {chartTitle}, type: {chartType}").format(chartTitle=name, chartType=chartTypeText)
+		count = self.wordChartObject.SeriesCollection().count

```

Comment:
Kept only name and title in _get_name method, rest of the code moved to _get_description method. Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.